quantum/process_keycode/process_leader.c

```diff
@@ -49,24 +49,24 @@ bool process_leader(uint16_t keycode, keyrecord_t *record) {
   // Leader key set-up
   if (record->event.pressed) {
     if (leading) {
-      if (timer_elapsed(leader_time) < LEADER_TIMEOUT) {
-#ifndef LEADER_KEY_STRICT_KEY_PROCESSING
+    //   if (timer_elapsed(leader_time) < LEADER_TIMEOUT) {
+        #ifndef LEADER_KEY_STRICT_KEY_PROCESSING
         if ((keycode >= QK_MOD_TAP && keycode <= QK_MOD_TAP_MAX) || (keycode >= QK_LAYER_TAP && keycode <= QK_LAYER_TAP_MAX)) {
           keycode = keycode & 0xFF;
         }
-#endif // LEADER_KEY_STRICT_KEY_PROCESSING
+        #endif // LEADER_KEY_STRICT_KEY_PROCESSING
         if ( leader_sequence_size < ( sizeof(leader_sequence) / sizeof(leader_sequence[0]) ) ) {
           leader_sequence[leader_sequence_size] = keycode;
           leader_sequence_size++;
         } else {
           leading = false;
           leader_end();
         }
-#ifdef LEADER_PER_KEY_TIMING
+        #ifdef LEADER_PER_KEY_TIMING
         leader_time = timer_read();
-#endif
+        #endif
```

Comment:
Please revert this formatting change:
```suggestion
#endif
```